src/components/vm/snapshots/vmSnapshotsCreateModal.jsx

```diff
@@ -91,6 +91,7 @@ const MemoryPathRow = ({ onValueChanged, memoryPath, validationError }) => {
             <FileAutoComplete
                 onChange={value => onValueChanged("memoryPath", value)}
                 superuser="try"
+                isOptionCreatable
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1435-20240209-093602-b1c5a2e1-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)